homework-7/ex-2/in.html

```diff
@@ -0,0 +1,145 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Open+Sans:wght@300&display=swap" rel="stylesheet">
+
+
+    <!-- Latest compiled and minified CSS -->
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/3.4.1/css/bootstrap.min.css" crossorigin="anonymous">
+
+    <!-- Optional theme -->
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/3.4.1/css/bootstrap-theme.min.css" crossorigin="anonymous">
+</head>
+<body>
+    <div class="container-fluid" style="background-color: #4e5366; height: auto; padding: 30px 0 0 0;">
+        <div class="col-md-1"></div>
+
+        <div class="col-md-10">
+            <nav class="navbar">
+                <div class="container-fluid">
+                    <div class="navbar-header">
+                        <a href="#" class="navbar-brand">
+                            <span style="color: #fff; font-size: 16px; border-right: 1px solid #aab0bd; padding-right: 10px;">ALTITUDU</span>
+                            <span style="color: #aab0bd; font-size: 12px; padding-left: 10px;">BY PIXELARITY</span>
+                        </a>
+                    </div>
+                    <div class="navbar-nav navbar-right" style="line-height: 50px; align-items: center;">
```

Comment:
Không được phép viết `inline style`
```suggestion
                    <div class="navbar-nav navbar-right">
```